src/ingestion/validators/polygonPartValidator.ts

```diff
@@ -69,9 +69,9 @@ export class PolygonPartValidator {
         logContext: logCtx,
         metadata: { polygonPart },
       });
-      throw new GeometryValidationError(polygonPart.sourceName as string, index, 'Geometry is invalid');
+      throw new GeometryValidationError(polygonPart.sourceName, index, 'Geometry is invalid');
     }
-    const containedByExtent = this.isContainedByExtent(polygonPart.geometry as Geometry, combinedExtent as GeoJSON);
+    const containedByExtent = this.isContainedByExtent(polygonPart.footprint as Geometry, combinedExtent as GeoJSON);
```

Comment:
footprint is from type "Polygon" why use `as Geometry` here?

Reply:
I just noticed the you need to convert it because 'containedBy' function accepts only 'Geometry', we need to think if it may make a troubles